.github/workflows/stale-docs-check.yml

```diff
@@ -47,22 +47,25 @@ jobs:
                           -e "docs-lakeFS/redirects.json" > /tmp/diffs.txt || true
 
           if [ -s "/tmp/diffs.txt" ]; then
-            echo "🚨 Docs published on docs.lakefs.io don't match docs generated from source 🚨"
-            echo " "
-            cat /tmp/diffs.txt
```

Comment:
Could we also keep this line?  It is often easier for me to open the failing step from the link on checks, and read the output directly on the step.